webapp/src/octoClient.ts

```diff
@@ -335,9 +335,17 @@ class OctoClient {
         })
     }
 
-    async undeleteBlock(blockId: string): Promise<Response> {
+    async undeleteBlock(boardId: string, blockId: string): Promise<Response> {
         Utils.log(`undeleteBlock: ${blockId}`)
-        return fetch(this.getBaseURL() + this.teamPath() + `/blocks/${encodeURIComponent(blockId)}/undelete`, {
+        return fetch(`${this.getBaseURL()}/api/v1/boards/${encodeURIComponent(boardId)}/blocks/${encodeURIComponent(blockId)}/undelete`, {
+            method: 'POST',
+            headers: this.headers(),
+        })
+    }
+
+    async undeleteBoard(boardId: string): Promise<Response> {
```

Comment:
Sorry - that doesn't make sense here.  I guess I'm confused about how someone undeletes a board and all the cards and blocks.

Reply:
Yes, now we don't have cascade deletion, you delete the board, is enough because the board info is not longer accesible (from the UI perspective), in the future we can decide to cascade that, or to interpret a block in a deleted board as deleted. But... For now, we only need the undelete of the board.